noggin/templates/user-settings-profile.html

```diff
@@ -2,7 +2,7 @@
 
 {% import '_form_macros.html' as macros %}
 {% block settings_content %}
-<form class="needs-validation" action="{{ url_for('user_settings_profile', username=user.username) }}" method="post" novalidate>
+<form class="needs-validation" id="profile-form" action="{{ url_for('user_settings_profile', username=user.username) }}" method="post" novalidate>
```

Comment:
Why set the id here?

Reply:
ha! was originally selecting by that ID, then stopped. this was just a leftover...